tools/tertiary-analysis/scmap/scmap_macros.xml

```diff
@@ -1,11 +1,12 @@
 <macros>
     <token name="@TOOL_VERSION@">1.6.0</token>
     <token name="@HELP@">More information can be found at https://bioconductor.org/packages/release/bioc/html/scmap.html</token>
+    <token name="@PROFILE@">18.01</token>
     <xml name="requirements">
       <requirements>
         <requirement type="package">openblas</requirement>
```

Comment:
This one will be my fault I think - but could you try removing the extraneous dependencies anyway?